### DISCUSSION TITLE
Welcome to ShimmysAdminTools Discussions!

### DISCUSSION THREAD

**Discussion by ShimmyMySherbet** [ShimmyMySherbet/ShimmysAdminTools#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by TeemoCell** [comment]
Just wanted to say thx for the amazing plugin.

**Comment by papershredder432** [comment]
bitch😔

**Reply by ShimmyMySherbet** [reply]
no u

**Comment by Texhammer** [comment]
Is there a log of what my staff /pt u? Because they dont have to type in a command for each item my concern is that there is no log of it and they can abuse it very easily.

**Reply by ShimmyMySherbet** [reply]
I've moved this to an issue (#24), to be added in the next update

**Reply by Texhammer** [reply]
Thanks!
